FAQ.md

```diff
@@ -0,0 +1,55 @@
+# Leela Zero常见问题解答 #
+# Frequently Asked Questions about Leela Zero #
+
+## 为什么网络不是每次都变强的 ##
+## Why doesn't the network get stronger every time ##
+
+从谷歌的论文中可以发现，AZ的网络强度也是有起伏的。而且现在只是在小规模测试阶段，发现问题也是很正常的。请保持耐心。
+
+AZ's network isn't always be stronger either. Besides we're actually testing our approach right now. Just be patient.
```

Comment:
Some grammar suggestions.

AZ also had this behavior, besides we're testing our approach right now. Please be patient.